api.js

```diff
@@ -195,7 +195,7 @@ function handlePaths(files) {
 		files = [
 			'test.js',
 			'test-*.js',
-			'test/*.js'
```

Comment:
> why isn't it safe to recurse the entire directory?

I just answered that: "This would match https://github.com/sindresorhus/ava/blob/master/lib/test.js" Which is not a test file.

> I have a really cool system, I can explain more if you like

Please do :)